diff --git a/node_modules/react-native-blob-jsi-helper/react-native-blob-jsi-helper.podspec b/node_modules/react-native-blob-jsi-helper/react-native-blob-jsi-helper.podspec
index 53ec31f..c47aba8 100644
--- a/node_modules/react-native-blob-jsi-helper/react-native-blob-jsi-helper.podspec
+++ b/node_modules/react-native-blob-jsi-helper/react-native-blob-jsi-helper.podspec
@@ -15,6 +15,10 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm}", "cpp/**/*.{h,cpp}"
 
+  s.pod_target_xcconfig = {
+    "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
+  }
+
   s.dependency "React-Core"
   s.dependency "React"
 end
